pages/design/rule-design.md

```diff
@@ -39,34 +63,56 @@ A test aspect is a part of a test subject that must be available in order to pro
 
 Other aspects may be necessary for testing. These can be added as long as they are sufficiently defined.
 
+For more details, see [ACT Rules Format: Input](https://www.w3.org/TR/act-rules-format/#input)
```

Comment:
I think ACT-Rules format should be cited upfront. Keep the individual parts where needed, but start by "the rule comply to ACT-Rule format" or something similar.

Reply:
Presume you mean move this sentence earlier in the page.